bin/network_profiling/labeldict_example.json

```diff
@@ -0,0 +1 @@
+{"0": ["willy-1", "0"], "1": ["willy-10", "0"], "2": ["willy-11", "0"], "3": ["willy-12", "0"], "4": ["willy-13", "0"], "5": ["willy-14", "0"], "6": ["willy-15", "0"], "7": ["willy-16", "0"], "8": ["willy-2", "0"], "9": ["willy-3", "0"], "10": ["willy-4", "0"], "11": ["willy-5", "0"], "12": ["willy-6", "0"], "13": ["willy-7", "0"], "14": ["willy-8", "0"], "15": ["willy-9", "0"], "16": ["willy-m", "0"], "17": ["9+10", "0.00003992"], "18": ["0+16", "0.00004864"], "19": ["11+12", "0.00005612"], "20": ["8+18", "0.00005702"], "21": ["4+5", "0.00005997"], "22": ["1+15", "0.00006612"], "23": ["6+7", "0.00006692"], "24": ["13+14", "0.00007658"], "25": ["2+3", "0.00009460"], "26": ["22+24", "0.01019850"], "27": ["19+26", "0.17401823"], "28": ["21+25", "0.18369587"], "29": ["23+28", "0.20939741"], "30": ["27+29", "2.24956349"], "31": ["17+20", "2.60744858"], "32": ["30+31", "5.40899999"]}
```

Comment:
how about moving example json files to `examples` folder?  Usually `bin/` dir includes only script files.